sleap/nn/training.py

```diff
@@ -177,14 +178,13 @@ def from_labels(
                     "Creating validation split from explicit indices "
                     f"(n = {len(labels_config.validation_inds)})."
                 )
-                validation = training[labels_config.validation_inds]
-
+                validation = Labels(training[labels_config.validation_inds])
```

Comment:
Here and in subsequent places, do we maybe want to switch to using [`Labels.extract()`](https://github.com/talmolab/sleap/blob/5093f6992e6214c0d528b7240331b99d0a89a62f/sleap/io/dataset.py#L780-L810)?

This retains other metadata which would be useful. We can skip using the `copy=True` since that might slow things down for really large datasets, but at least using `extract()` will avoid some weird issues with indexing videos and etc.

We may also want to switch to `extract()` here in `sleap.nn.data.training.split_labels_train_val()` for consistency:

https://github.com/talmolab/sleap/blob/5093f6992e6214c0d528b7240331b99d0a89a62f/sleap/nn/data/training.py#L50-L59